test/azure/rules.spec.ts

```diff
@@ -1,33 +1,42 @@
 import {assert} from 'chai';
-import {Rule, ResourceGraphRule} from '../../src/rules';
+import {
+  ResourceGraphRule,
+  ResourceGraphExecutor,
+  ResourceGraphTarget,
+  RuleType,
+} from '../../src/rules';
 import {subscriptionId} from '.';
 
 describe('Resource Graph Rule', function () {
-  this.slow(5000);
-  this.timeout(8000);
+  this.slow(6000);
+  this.timeout(10000);
 
   it('can execute a resource graph rule and return a scan result', async () => {
     const query =
       "Resources | where type =~ 'Microsoft.Compute/virtualMachines2'";
-    const name = 'Dummy Rule';
+    const name = 'test-rule';
```

Comment:
nit: if these are all single-use, you could inline them all into the rule creation below to make it easier to scan/read

Reply:
ok, that makes more sense.